src/components/Input/Input.tsx

```diff
@@ -1,22 +1,12 @@
 import styles from './Input.module.css';
-import { ChangeEvent } from 'react';
+import { ChangeEvent, ComponentProps } from 'react';
 
 type Props = {
   value: string;
-  placeholder: string;
-  isValid: boolean;
+  isError: boolean;
   onChange: (e: ChangeEvent<HTMLInputElement>) => void;
-  maxLength: number;
-};
+} & ComponentProps<'input'>;
 
-export default function Input({ value, placeholder, isValid, onChange, maxLength }: Props) {
-  return (
-    <input
-      value={value}
-      onChange={onChange}
-      placeholder={placeholder}
-      className={`${styles.input} ${isValid ? styles.valid : styles.inValid}`}
-      maxLength={maxLength}
-    />
-  );
+export default function Input({ value, isError, onChange, ...props }: Props) {
```

Comment:
```suggestion
export default function Input({ isError, ...props }: Props) {
```
`value` 속성과 `onChange` 속성은 `ComponentProps<'input'>` 내장되어 있기 때문에 별도의 타입 선언이 필요없을 것 같고 `...props`와 같이 rest parameters에 포함시킬 수 있을 것 같은데 따로 분리해주신 이유가 있으실까요??🤔

Reply:
딱히 이유는 없고 처음에는 ComponentProps를 사용할 생각을 못했어서 그런 거 같아요! ! 수정했습니당